### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andrianvaleanu**
Fixed suggestions from previous commit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +90,6 @@ Total deals: 110
 ### 💬 AI Chat UI
 |  | Name | Description | Discount Code & Terms |
 | -- | ---| ------ | ------ |
-| 🧠 | [TypingMind](https://www.typingmind.com/?utm_source=tonybf) | Personal AI workspace. Frontend for ChatGPT, Claude, Gemini to use with your API key. Support Artifact, Projects, Vision, Canvas, AI agents builder, DALL-E, Plugins, etc. | 70% OFF LIFETIME **BLACKFRIDAY_2024** |
```
Comment (dqhieu) [inline README.md:91]: Is this intended?

File: README.md [hunk 2]
```diff
@@ -80,6 +80,8 @@ Total deals: 110
 |  ⭐ | [(Tailscan for Tailwind CSS)](https://tailscan.com) | The ultimate devtool browser extension for Tailwind CSS. Inspect, debug, convert and more! | 25% OFF! |
 |  ⭐ | [snappify](https://snappify.com) | A powerful PowerPoint alternative for developers. Create stunning presentations, smoothly animated videos and elevate your technical content! | 30% OFF the first year with the code **BF_2024** |
 | 📸 | [ScreenshotOne](https://screenshotone.com) | The best screenshot API for developers. | 25% OFF with code **BLACKFRIDAY2024** on all annual plans for the first year. |
+|  ⭐ | [(Static.app](https://static.app/) | Static website hosting - One-click hosting solution for your static website. | 30% OFF with code **BLACK24** |
```
Comment (dqhieu) [inline README.md:83]: The link is not accessible

Comment (andrianvaleanu) [reply README.md:83]: The link is live now, the wbesite has been updated.

File: README.md (hunk 1 above)
Comment (andrianvaleanu) [reply README.md:91]: Nope, it's an issue.

Comment (dqhieu) [reply README.md:91]: Can you please reverse this change?

**Review by andrianvaleanu — COMMENTED** [review]
Finished the review and fixed.

File: README.md (hunk 1 above)
Comment (andrianvaleanu) [reply README.md]: Done.

**Pull request merged**